### PR TITLE
Membership Saves - Create Switching Page [Initial design]

### PR DESCRIPTION
**Pull request by andrewHEguardian**
## What does this change?

Create the initial design of the Switching Page in new Membership save journey.

## How to test

Added to storybook, not yet hooked up to any routes.

## Images

image

## Accessibility

-   [ ] [Tested with screen reader](https://github.com/guardian/accessibility/blob/main/people-and-technology/03-visual.md#screen-reader)
-   [ ] [Navigable with keyboard](https://github.com/guardian/accessibility/blob/main/people-and-technology/02-physical.md#keyboard)
-   [ ] [Colour contrast passed](https://github.com/guardian/accessibility/blob/main/people-and-technology/03-visual.md#contrast)
-   [ ] [The change doesn't use only colour to convey meaning](https://github.com/guardian/accessibility/blob/main/people-and-technology/03-visual.md#use-of-colour)

### CODE REVIEW

File: client/components/mma/cancel/cancellationSaves/MembershipSwitch.tsx [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+				<p
+					css={css`
+						${textSans.medium()}
+						border-top: 1px solid ${palette.neutral[86]};
+						padding-top: ${space[5]}px;
+					`}
+				>
+					Please note that, once the change is done, you will not be
+					able to have the full set of benefits for £7 again (full
+					price is now £10).
```
Comment (charleycampbell) [inline client/components/mma/cancel/cancellationSaves/MembershipSwitch.tsx:185]: Do we need to add the part about they can't have this offer again?

**Review by charleycampbell — COMMENTED** [review]
Looks good just one comment re wording that's on the wire frame but otherwise looks good! 👍

**Comment by andrewHEguardian** [comment]
[Membership Saves - Switching Page [Initial design]](https://trello.com/c/PAjDIGU5/4039-membership-saves-switching-page-initial-design)

File: client/components/mma/cancel/cancellationSaves/MembershipSwitch.tsx (hunk 1 above)
Comment (andrewHEguardian) [reply client/components/mma/cancel/cancellationSaves/MembershipSwitch.tsx:185]: let's come back to this when we have final copy, that seemed like a very rough draft

**Review by charleycampbell — APPROVED** [review]
Nice 👍

**Pull request merged**